app/src/components/features/rules/RulesListContainer/RulesTable/index.js

```diff
@@ -612,7 +612,9 @@ const RulesTable = ({
               }}
             >
               <Link
+                href="/"
```

Comment:
`href="/"` means link to the root path. If you open this link in new tab, it is going to load the root path of the app which is the Rules list page itself.

The expectation was to open the rule definition in the new tab - that is, it should open the same screen that opens up on normal clicking of the link.

Reply:
Thanks for the feedback. I'll fix the issue.

Reply:
Hey @lazyvab updated the href path from "/" to {`${PATHS.RULE_EDITOR.EDIT_RULE.ABSOLUTE}/${record.id}`}. I hope I met the expectation.